File: no-auth/src/main/java/com/kjj/noauth/util/jwt/JwtTemplate.java
package com.kjj.noauth.util.jwt;

import org.springframework.stereotype.Component;

@Component
public class JwtTemplate {
    public String getSecret() { return "capston2023kjj";}

    public int getExpiration() {
        return 30;
    }

    public String getTokenPrefix() {
        return "Bearer ";
    }

    public String getHeaderString() {
        return "Authorization";
    }

    public String getRefreshHeaderString() {
        return "Refresh_Token";
    }

    public String getRefreshType() { return "Refresh"; }
}

File: no-auth/src/main/kotlin/com/kjj/noauth/tool/jwt/JwtTemplate.kt
package com.kjj.noauth.tool.jwt

import org.springframework.stereotype.Component

@Component
class JwtTemplate(
    val secret: String = "capston2023kjj",
    val expiration: Long = 30,
    val tokenPrefix: String = "Bearer ",
    val headerString: String = "Authorization",
    val refreshHeaderString: String = "Refresh_Token",
    val refreshType: String = "Refresh"
    ) {
}